packages/bs-ethereum/src/BlockscoutNeoXBDSEthereum.ts

```diff
@@ -301,8 +337,12 @@ export class BlockscoutNeoXBDSEthereum extends RpcBDSEthereum {
       throw new Error('Token not found')
     }
 
+    if (data.type !== 'ERC-20') {
+      throw new Error('Token is not an ERC-20 token')
+    }
+
     return {
-      decimals: parseInt(data.decimals),
+      decimals: data.decimals ? parseInt(data.decimals) : 18,
```

Comment:
Wouldn't it be nice if you put "18" in an explainable variable like `DEFAULT_DECIMALS`?